### DISCUSSION TITLE
Not received confirmation email for copilot global event

### DISCUSSION THREAD

**Discussion by amit-jesani** [microsoft/global-copilot-summit#2] [General Q&A]
I register myself for Mumbai event but not received any confirmation this event.

**Comment by kamalshree** [comment]
Hi Amit,

We will rolling out the confirmation email shortly.

**Reply by saisuseelmohan2003** [reply]
Mam, i registered for the event today, and didn't get any reply. Please help 😭

**Reply by amit-jesani** [reply]
@kamalshree 
Tomorrow is event still not received any mail for confirmation

My team member is also applied but still not received any confirmation mail.

**Comment by heliumcargo1** [comment]
i applied for the Hyderabad summit, not yet received the confirmation email.
